src/test/java/bio/terra/service/rawls/RawlsServiceTest.java

```diff
@@ -112,32 +90,26 @@ public void testRetrievePoliciesAndEmailsWorkspace() {
             .memberPolicies(resourcePolicyModels);
 
     var workspacePolicyModels = rawlsService.resolvePolicyEmails(samPolicyModel, TEST_USER);
-    verify(rawlsClient).getWorkspace(accessibleWorkspaceId, TEST_USER);
-    verify(rawlsClient).getWorkspace(inaccessibleWorkspaceId, TEST_USER);
 
     assertThat(
         "Workspace accessible to the user is included in returned policy models",
         workspacePolicyModels.accessible(),
```

Comment:
There is a way to assert multiple things about a single object, if you wanted to write this that way. https://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/core/AllOf.html
```
assertThat(workspacePolicyModels.accessible(), allOf(contains(...), contains(...)));
```

Although in this specific case I think `hasItems()` could work too.